beaker-pe.gemspec

```diff
@@ -32,6 +32,7 @@ Gem::Specification.new do |s|
 
   # Run time dependencies
   s.add_runtime_dependency 'stringify-hash', '~> 0.0.0'
+  s.add_runtime_dependency 'in-parallel', '~> 0.1'
```

Comment:
Now that beaker itself has a runtime dependency requires `in-parallel`, does it make sense for beaker-pe to also specify the version? I think it would be cleaner to manage the the actual version in beaker itself, and then leave the dependency here with an unspecified version number, but with a comment about where that version is managed. What do you think?